ndk-build/src/manifest.rs

```diff
@@ -277,6 +246,11 @@ pub struct Feature {
     pub name: Option<String>,
     #[serde(rename(serialize = "android:required"))]
     pub required: Option<bool>,
+    /// When `name` is FEATURE_VULKAN_HARDWARE_COMPUTE, see: https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_COMPUTE
+    /// When `name` is FEATURE_VULKAN_HARDWARE_LEVEL, see: https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_LEVEL
+    /// When `name` is FEATURE_VULKAN_HARDWARE_VERSION, see: https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_VERSION
```

Comment:
Iirc you're supposed to place links `<https://developer.android.com/reference/android/content/pm/PackageManager#FEATURE_VULKAN_HARDWARE_COMPUTE>` but please double-check the documentation.

It looks like we only test/check the docs now, perhaps need a `cargo test --doc` in a separate PR (perhaps prior to this PR).

Reply:
You should perhaps write out the _string values_ for these constants, so `"android.hardware.vulkan.level"` and the like. Perhaps also explain in short what the value means, before referring to the Android docs (those links like to break every year or so).